### PR TITLE
behat -- adding an ASP community persona test

### PR DESCRIPTION
**Pull request by jasperlieber**
## Describe context / purpose for this PR
Adding an ASP community persona test.  This now includes test data from amp_dev.install. This change includes:

update users 200 and 201 to have some specified values, esp. roles, affinity groups, academic status, interests, skills
create 2 ci-links for those 2 users
The behat test community_persona.feature then checks for these values.

Note:  I also wanted to create some dummy engagements and projects, but run into trouble doing that.  I want to push what I have so far because I'm gone next week and I need to spend some time on other items, like bigger behat questions & training the interns.

## Issue link
https://cyberteamportal.atlassian.net/browse/D8-1421

## Any other related PRs?
https://github.com/necyberteam/amp_dev/pull/1/

## Link to MultiDev instance

http://md-{{ ticket number }}-accessmatch.pantheonsite.io

## Checklist for PR author
- [x] I have checked that the PR is ready to be merged
- [x] I have reviewed the DIFF and checked that the changes are as expected
- [x] I have assigned myself or someone else to review the PR

### CODE REVIEW

File: tests/behat/features/asp/community_persona-unauth.feature [hunk 1]
```diff
@@ -0,0 +1,59 @@
+@asp
+@api
+@javascript
+
+Feature: test ACCESS Support Community Page
+  In order to test the Community Page
```
Comment (a-pasquale) [inline tests/behat/features/asp/community_persona-unauth.feature:6]: Community -> Community Persona

File: tests/behat/features/asp/community_persona-unauth.feature [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  Scenario: Unauthenticated user tests the Community Page for user 1
+    Given I am not logged in
+    When I go to "/community-persona/1"
+    And I wait for the page to be loaded
+    Then I should see "Login to Create Ticket"
```
[the remark this answers is not in the document]
Comment (jasperlieber) [reply tests/behat/features/asp/community_persona-unauth.feature]: I remove it

**Comment by jasperlieber** [comment]
@a-pasquale -- I updated the main description of this PR.  I learned *a lot* about creating test data, and the community-persona.feature test is now more robust.  But it could have more improvements -- for example, I was not able to create dummy engagements or projects, so these aren't specifically tested.  

(I'm also not sure about the conflict in composer.lock -- maybe that's simple, but I'm out of time to figure it out.)

**Comment by jasperlieber** [comment]
redone on https://github.com/necyberteam/cyberteam_drupal/pull/769

**Pull request closed without merging**